src/module/helpers/rolls.mjs

```diff
@@ -509,3 +509,43 @@ export class ProjectRoll extends DSRoll {
     return context;
   }
 }
+
+export class SavingThrow extends DSRoll {
+  /**
+   * @param {string} [formula="1d10"]        Default saving throw is a flat 1d10
+   * @param {Record<string, any>} [data]     Roll data
+   * @param {{ flavor?: string }} [options]  Options to modify roll display
+   */
+  constructor(formula = "1d10", data = {}, options = {}) {
+    super(formula, data, options);
+  }
```

Comment:
Do we foresee any instance of the formula being anything but `1d10`?

Reply:
It looks like there are some effects that can give a bonus to saving throws, found a high level Censor feature that gives a +2 bonus to them.

Reply:
> It looks like there are some effects that can give a bonus to saving throws, found a high level Censor feature that gives a +2 bonus to them.

OK. Maybe it would be good to have a follow-up PR for constructing the roll's different parts; with bonuses, whichever modifiers, etc.

Reply:
Yeah, once we write functions like "use an ability" I'm sure we'll realize new ergonomics are needed